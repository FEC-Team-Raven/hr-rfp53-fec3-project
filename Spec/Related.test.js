/* eslint-disable no-undef */
const Related = require('../client/src/components/related/Related');

test('', () => {
  expect('');
})