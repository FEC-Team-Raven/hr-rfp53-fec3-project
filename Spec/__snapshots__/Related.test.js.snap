// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and adds outfit 1`] = `
<div>
  <div
    class="outfit grid-container"
  >
    <div
      class="card addOutfit"
    >
      <button>
        +
      </button>
      <h2>
        Add to Outfit
      </h2>
    </div>
  </div>
</div>
`;
